cmd/server/app/migrate_up.go

```diff
@@ -127,10 +130,64 @@ var upCmd = &cobra.Command{
 			return fmt.Errorf("error preparing authz client: %w", err)
 		}
 
-		return nil
+		store := db.NewStore(dbConn)
+		return migrateProjectsRootAndMetadata(ctx, cmd, store)
 	},
 }
 
+func migrateProjectsRootAndMetadata(ctx context.Context, cmd *cobra.Command, store db.Store) error {
+	cmd.Println("Migrating projects to have no parent...")
+	projs, err := store.ListNonOrgProjects(ctx)
+	if err != nil {
+		return fmt.Errorf("error while listing non-org projects: %w", err)
+	}
+
+	for _, p := range projs {
+		m := projects.NewSelfEnrolledMetadata()
```

Comment:
Can we move the inside of this loop into a function, as a general practice? It seems like that would make testing easier.  I'm not advocating for testing this yet, but let's start structuring our code around testability for cases like this.